app/themes/standard/views/comments/_comment_form_introduction.html.haml

```diff
@@ -7,7 +7,9 @@
     or one of your elected local councillors.
   - else
     Your comment and details will be sent to
-    %strong #{application.authority.full_name}.
-    - unless application.official_submission_period_expired?
-      They’ll consider your submission when they decide whether to approve this application.
+    %strong #{application.authority.full_name}, 
+    - if application.official_submission_period_expired?
+      but cannot be formally considered by them as the notification period for this application has expired.
+    - else
+       and may be considered by them as part of the decision making process for this application.
```

Comment:
Why not simply replace "They'll consider" with "They may consider"? I don't like the use "decision making process" because it's passive bureaucratic language. The authority is deciding whether to approve the application. That's more active language.